packages/extension-chakra-storefront/src/components/field/index.jsx

```diff
@@ -60,6 +60,8 @@ const Field = ({
                 const _inputProps =
                     typeof inputProps === 'function' ? inputProps({value, onChange}) : inputProps
 
+                const {endElement, ...restOfInputProps} = _inputProps || {}
```

Comment:
This will address the warning.
```
console.error
    Warning: React does not recognize the `endElement` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `endelement` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```